src/styles/utilities/_spacers.scss

```diff
@@ -1,11 +1,26 @@
 @use "../abstracts" as *;
 
+/**
+ * Padding
+ */
+
+// Top / Bottom
+$py: (
+	"48": 4.8rem,
+);
+@include generate-utils("py", ("padding-top", "padding-bottom"), $py);
+
 /**
  * Margin
  */
 
-// Right
+// Left / Right
+$ml: (
+	"8": $spacer-xs,
+);
+@include generate-utils("ml", "margin-left", $ml);
+
 $mr: (
 	"8": $spacer-xs,
 );
```

Comment:
Bij `$spacer`-utils ben ik geneigd om de suffix als key te gebruiken ipv de waarde. `u-ml-xs` ipv `u-ml-8`

Reply:
Ik heb voor de waarde (in px) geopteerd zodat je direct weet wat het is zonder nog naar de variables te moeten kijken.